app/assets/stylesheets/frontend/views/_person-profile.scss

```diff
@@ -0,0 +1,18 @@
+.person-profile {
+  border-top: 1px solid $govuk-border-colour;
+  padding-top: govuk-spacing(4);
+  margin-bottom: govuk-spacing(6);
+}
+
+.person-profile__image {
```

Comment:
Is there a variation of the image card that can be used here instead

Reply:
There isn't. I'm not sure about extending an existing component for the sake of this one as I don't think there's necessarily harm in a tiny pattern that requires zero customisation existing in the only app it's being used in. This pattern exists for `app-c` components across our other frontend apps ( I can try `app-c`ing this if you'd like). What do you think?

Reply:
Yeah I think `app-c`-ing is a good shout, however in order of preference:

- Use a current version of image card here (if possible change the design to match)
- `app-c`-ing as this types of image use are Whitehall-centric from what I can tell
- Add a new variation to cater for this new Design so it can be reused elsewhere.

Reply:
Reviewing this, there isn't a good `image_card` that would sit here.  I considered extended this the component itself.

However, taking a step back and looking at the wider page context, this image is decorative, the main page title tells the user what this is.  In this context i'm proposing a good solution is just to use an image tag with an empty alt